lisa/search_space.py

```diff
@@ -710,3 +710,15 @@ def create_set_space(
     else:
         set_space = None
     return set_space
+
+
+def decode_empty_count_space(
```

Comment:
It should be called like `decode_nullable_set_space`. It's not a count space or always empty one.